Xabe.FFmpeg/FFmpeg.cs

```diff
@@ -72,6 +74,30 @@ public static Task GetLatestVersion(bool fullVersion = false)
                 downloader = new FFmpegDownloader();
             }
             return downloader.GetLatestVersion();
+        }
+
+        /// <summary>
+        ///     Download latest FFmpeg version for current operating system to FFmpeg.ExecutablePath. If it is not set download to ".".
+        /// <param id="version">Determine which version of FFmpeg should be downloaded</param>
+        /// </summary>
+        public static Task GetLatestVersion(FFmpegVersions version)
```

Comment:
FFmpegVersions change this to FFmpegVersion and I will merge that :)

Reply:
Done